### PR TITLE
DataTable demo

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Review by pawellysy — CHANGES_REQUESTED** [review]
Could you please add a description, how to use this demo?

**Review by goransle — COMMENTED** [review]
Think I found a bug: the second time I add a column it also adds an unnamed column
image

File: ts/masters/modules/data-tools.src.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/**
+ * @license Highcharts JS v@product.version@ (@product.date@)
+ * @module highcharts/modules/data-tools
```
Comment (karolkolodziej) [inline ts/masters/modules/data-tools.src.ts:3]: Is it temporary or is it going to be an official module?

**Review by karolkolodziej — APPROVED** [review]

File: ts/masters/modules/data-tools.src.ts (hunk 1 above)
Comment (bre1470) [reply ts/masters/modules/data-tools.src.ts:3]: Temporary.

**Review by pawelfus — APPROVED** [review]
Thanks!

Q: `setRow()` does not add the row somewhere in the middle - always at the end? Even for `modified` table? For example I have `sort` modifier -> add a row -> do I need to sort again?

**Comment by bre1470** [comment]
@pawelfus Good point. It should add the row to the end of the original table, and then the sorted one should sort again. I forgot to trigger the modifier.

**Review by pawellysy — APPROVED** [review]
LGTM!

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Review by goransle — APPROVED** [review]

**Pull request merged**